lld/Common/Filesystem.cpp

```diff
@@ -57,7 +58,7 @@ void lld::unlinkAsync(StringRef path) {
   //
   // The code here allows LLD to work on all versions of Windows.
   // However, at Windows 10 1903 it seems that the behavior of
-  // Windows has changed, so that we could simply delete the output 
+  // Windows has changed, so that we could simply delete the output
```

Comment:
Nit: Stray change

(The cleanup of trailing whitespace certainly is good though, but it'd be better to keep it in a separate commit, which I guess you could push just bypassing review.)